website/docs/cli/import/index.mdx

```diff
@@ -9,72 +9,16 @@ description: >-
 
 > **Hands-on:** Try the [Import Terraform Configuration](https://learn.hashicorp.com/tutorials/terraform/state-import?in=terraform/state&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.
 
-<<<<<<< HEAD
```

Comment:
I'm really sorry about all of this. It's the result of me accidentally prematurely merging my changes into main and then trying to revert them :(. Compare my edits with the original page here: https://developer.hashicorp.com/terraform/cli/import